src/pages/Filmography.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+
+import {
+  Route,
+  HashRouter,
+} from "react-router-dom";
+
+import FilmographyDetail from "../components/templates/FilmographyDetail"
+import FilmographyList from "../components/templates/FilmographyList"
+
+import moviePoster from "../assets/no-img.png"
+
+
+// const {Provider, Consumer} = React.createContext()
+
+class Filmography extends Component {
+  state = {
+    movies: []
+  }
+  componentDidMount(){
+    const apiUrl = 'https://ghibliapi.herokuapp.com/films'
+    fetch(apiUrl)
```

Comment:
for now, it is okay to use built-in fetch method. But later try to read about `axios`.

And also avoid `.then` callbacks. Read more about **Callback hell in javascript**

@alramdein